op_builder/mlu/fused_adam.py

```diff
@@ -5,7 +5,11 @@
 # DeepSpeed Team
 
 from .builder import MLUOpBuilder
-import torch
```

Comment:
FYI @Andy666G - this change broke the no-torch workflow.  See more info in the PR comment, but please let us know if you have any concerns with this change,